File: mobile/Flutter/lib/Models/jouet.dart
class Jouet {
  String? _id;
  String? _titre;
  String? _description;
  String? _etatJouet;
  String? _categorieJouet;

  Jouet(this._id, this._titre, this._description, this._etatJouet, this._categorieJouet);

  String? get categorieJouet => _categorieJouet;

  String? get id => _id;

  set id(String? value) {
    _id = value;
  }

  set categorieJouet(String? value) {
    _categorieJouet = value;
  }

  String? get etatJouet => _etatJouet;

  set etatJouet(String? value) {
    _etatJouet = value;
  }

  String? get description => _description;

  set description(String? value) {
    _description = value;
  }

  String? get titre => _titre;

  set titre(String? value) {
    _titre = value;
  }
}

File: mobile/android/app/src/main/java/com/example/exchange_and_donate/entities/Jouet.java
package com.example.exchange_and_donate.entities;

public class Jouet {
    private String id ;
    private String titre ;
    private String description ;
    private String etat_jouet ;
    private String categorie_jouet ;
    private String uploaded_by;
    private Boolean donate  ;

    public Jouet(String id, String titre, String description, String etat_jouet, String categorie_jouet, String uploaded_by, Boolean donate) {
        this.id = id;
        this.titre = titre;
        this.description = description;
        this.etat_jouet = etat_jouet;
        this.categorie_jouet = categorie_jouet;
        this.uploaded_by = uploaded_by;
        this.donate = donate;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getEtat_jouet() {
        return etat_jouet;
    }

    public void setEtat_jouet(String etat_jouet) {
        this.etat_jouet = etat_jouet;
    }

    public String getCategorie_jouet() {
        return categorie_jouet;
    }

    public void setCategorie_jouet(String categorie_jouet) {
        this.categorie_jouet = categorie_jouet;
    }

    public String getUploaded_by() {
        return uploaded_by;
    }

    public void setUploaded_by(String uploaded_by) {
        this.uploaded_by = uploaded_by;
    }

    public Boolean getDonate() {
        return donate;
    }

    public void setDonate(Boolean donate) {
        this.donate = donate;
    }
}

